tests/e2e/unified_test.go

```diff
@@ -81,7 +83,33 @@ func TestAllInSeries(t *testing.T) {
 		}
 	})
 
-	t.Run("fixtures", func(t *testing.T) {
+	testFixturesInSeries(ctx, t, "AllInSeries", false, cancel)
+}
+
+// TestPauseInSeries is a basic smoke test to prove that if CC pauses actuation of resources
+// via the actuationMode field, then resources are not actuated onto the cloud provider.
+// The current test is to make sure that POST requests are not recorded as HTTP events.
```

Comment:
More complex scenarios like "pause then unpause" or "pause" in a specific namespace should follow. They can use some of the step ideas from #1290

Reply:
Cool - I needed the step test in #1360 (and that one is a 1.114 blocker I believe), so hopefully we'll some of the stepwise test infrastructure in today!